pipelines/batch/src/main/java/org/openmrs/analytics/FetchResources.java

```diff
@@ -32,6 +35,7 @@
 import org.hl7.fhir.r4.model.Property;
 import org.hl7.fhir.r4.model.Reference;
 import org.hl7.fhir.r4.model.Resource;
+import org.joda.time.Instant;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
```

Comment:
nit: Update comment above class as it currently says:
`with 1s as the value to make it easy to count in downstream functors`

Reply:
Done.